packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.3.1
+
+* Enable mocking models by Changing overridden operator == parameter type from `dynamic` to `Object`
```

Comment:
Nit: Enables
Nit: Missing period at end.

Please see the CHANGELOG style guide linked from the PR description.